.vscode/extensions.json

```
@@ -6,6 +6,7 @@
     "rust-lang.rust-analyzer",
     "serayuzgur.crates",
     "tamasfe.even-better-toml",
-    "vadimcn.vscode-lldb"
+    "vadimcn.vscode-lldb",
+    "usernamehw.errorlens"
```

To keep this alphabetic:

```suggestion
    "usernamehw.errorlens",
    "vadimcn.vscode-lldb"
```